reference/storage/file-signUrl.md

```diff
@@ -0,0 +1,60 @@
+Create a signed url for access to file
+
+```javascript
+import { bucket, FileMode } from '@nitric/sdk';
+
+const assets = bucket('assets').for('reading', 'writing');
+
+const logo = assets.file('images/logo.png');
+
+const logoUrl = await logo.signUrl(FileMode.Read);
```

Comment:
A comment here to highlight the 'read' mode could be useful.

Reply:
done